library/src/morpheus-master/sass/_defaults.scss

```diff
@@ -165,6 +165,9 @@ $skin-with-icons: true !default;
 $icon-size: 20px !default;
 $tool-menu-icon-on-left: true !default;
 
+/* Icon Overrides */
```

Comment:
Yeah generally someone would put any of the values they would like to change in the customizations file. I was just documenting it here as the default icons are all those we were previously configured.